glide.lock

```diff
@@ -206,6 +213,8 @@ imports:
   - internal/log
   - internal/modules
   - internal/remote_api
+- name: google.golang.org/appengine/datastore
```

Comment:
Need to pin dependencies whose commits SHA have changed?

Reply:
This one is already pinned in glide.yaml, not sure why it's not showing up in old glide.lock, gonna pin the other dependencies.